src/pages/en/community/index.mdx

```diff
@@ -0,0 +1,48 @@
+---
+layout: "@layouts/BaseLayout.astro"
+title: Community
+---
+
+import { HeaderLogo, Footer, CommunityBanner, CommunityGridCard, ContentSection, HeroImage} from "@components";
+
+<HeroImage/>
```

Comment:
Can you remove the Hero element from the community page?

Reply:
removed for the next commit!